tests/Unit/Evolution/Systems/RadiationTransport/M1Grey/Test_M1Closure.cpp

```diff
@@ -0,0 +1,90 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <array>
+#include <cstddef>
+#include <utility>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/EagerMath/DeterminantAndInverse.hpp"
+#include "DataStructures/Tensor/EagerMath/DotProduct.hpp"
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"
+#include "DataStructures/Tensor/IndexType.hpp"  // IWYU pragma: keep
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/Systems/RadiationTransport/M1Grey/M1Closure.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/Gsl.hpp"
+
+/// Test M1 closure function
+SPECTRE_TEST_CASE("Evolution.Systems.RadiationTransport.M1Grey.M1Closure",
```

Comment:
Is there a test for the small velocity limit? I don't know enough about the code to be able to tell :)

Reply:
Not at the moment - do you mind if I add this (and possibly randomized test, if it's manageable) as a second PR? I'd like to at least merge the directory structure, so that I can work on the rest of the M1 code.

Reply:
Sure, that's fine by me!